<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>scheduleScanObject</class>
 <widget class="QDialog" name="scheduleScanObject">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>445</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>445</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>445</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Scheduled Scan-Job</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="picLabel">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../resources.qrc">:/icons/icons/media.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="headerLabel">
       <property name="font">
        <font>
         <pointsize>24</pointsize>
        </font>
       </property>
       <property name="text">
        <string>.... scheduled Scan-Job</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>188</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="scanJobHeader">
     <property name="styleSheet">
      <string notr="true">padding:5px;background:#c0c0c0</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>Scan-Job : </string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Log-Messages / Status</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QPlainTextEdit" name="logMessagePlainTextEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="currentFileLabel">
       <property name="styleSheet">
        <string notr="true">padding:5px;background:yellow</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string>Status : </string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>388</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="stopButton">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>stop</string>
       </property>
       <property name="icon">
        <iconset resource="../resources.qrc">
         <normaloff>:/icons/icons/stop.png</normaloff>:/icons/icons/stop.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>close</string>
       </property>
       <property name="icon">
        <iconset resource="../resources.qrc">
         <normaloff>:/icons/icons/application-exit.png</normaloff>:/icons/icons/application-exit.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>1</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="engineVersionLabel">
          <property name="styleSheet">
           <string notr="true">background:#b0b0b0;padding:5px</string>
          </property>
          <property name="text">
           <string>Engine Version : </string>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="scannedDirectoriesLabel">
          <property name="styleSheet">
           <string notr="true">background:#b0b0b0;padding:5px</string>
          </property>
          <property name="text">
           <string>Scanned Directories : </string>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="scannedFilesLabel">
          <property name="styleSheet">
           <string notr="true">background:#b0b0b0;padding:5px</string>
          </property>
          <property name="text">
           <string>Scanned Files : </string>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="errorsLabel">
          <property name="styleSheet">
           <string notr="true">background:#b0b0b0;padding:5px</string>
          </property>
          <property name="text">
           <string>Total Errors : </string>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="infectedFilesLabel">
          <property name="styleSheet">
           <string notr="true">background:#b0b0b0;padding:5px;</string>
          </property>
          <property name="text">
           <string>Infected Files : </string>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>stopButton</sender>
   <signal>clicked()</signal>
   <receiver>scheduleScanObject</receiver>
   <slot>slot_stopButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>583</x>
     <y>390</y>
    </hint>
    <hint type="destinationlabel">
     <x>457</x>
     <y>348</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>scheduleScanObject</receiver>
   <slot>slot_closeButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>689</x>
     <y>390</y>
    </hint>
    <hint type="destinationlabel">
     <x>695</x>
     <y>353</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>errorsLabel</sender>
   <signal>clicked()</signal>
   <receiver>scheduleScanObject</receiver>
   <slot>slot_totalErrorButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>489</x>
     <y>420</y>
    </hint>
    <hint type="destinationlabel">
     <x>499</x>
     <y>441</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>infectedFilesLabel</sender>
   <signal>clicked()</signal>
   <receiver>scheduleScanObject</receiver>
   <slot>slot_infectedFilesButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>634</x>
     <y>415</y>
    </hint>
    <hint type="destinationlabel">
     <x>695</x>
     <y>426</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slot_stopButtonClicked()</slot>
  <slot>slot_closeButtonClicked()</slot>
  <slot>slot_infectedFilesButtonClicked()</slot>
  <slot>slot_totalErrorButtonClicked()</slot>
 </slots>
</ui>
